recipes/deepforeset/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "DeepForest" %}
+{% set version = "0.2.17" %}
+{% set sha256 = "b03dad820445814df03f67960b997a125e2a6861be74c7221565149cebcd11f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py==37]
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - set SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [win]
+    - {{ PYTHON }} -m pip install --no-deps .
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - setuptools_scm
+    - pip >=20.1
+    - numpy >=1.14.0
+    - cython >=0.28
+
+  run:
+    - python
+    - matplotlib
+    - sphinx
+    - recommonmark
+    - pyyaml
+    - pytest
+    - numpydoc
+    - cython >=0.28
```

Comment:
Do you need cython at runtime?

Reply:
I think we do, I will confirm form the project lead @bw4sz

Reply:
cython is needed to build the keras-retinanet anchors. I think it is safest to have it around, we could try to navigate something if this is a large obstacle.

Reply:
But cython should just be needed to build, unless you are shipping something that you can `cimport` in a downstream package